README.md

```diff
@@ -1,55 +1,52 @@
-# HWSC-File-Transaction-SVC
-This is one of the microservices in the Humpback Whale Social Call project, which allows user to manage their files. 
-### Purpose
+# hwsc-file-transaction-svc
+This manages HWSC user file transactions
 
+### Purpose
 This provides file management service which includes the following:
-  - Downloading
-  - Uploading
-  - Deleting
+- DownloadFile()
+- UploadFile()
+- DeleteFile()
+
 ## Prerequisites
-- Python 3.7
-- Webstorm
+- Python 3.7 or greater
+- pip
+- pipenv
+- Pycharm or VSCode
+
+## Check your python version
+- Open your terminal and type `$ python --version`
+
+## How to install Python3.7 with Homebrew
+- On your terminal, type `$ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"` 
```

Comment:
if it's a super long command use triple back ticks. This will keep it in one line and users can choose to scroll to see the rest. The triple back ticks have to be in its own line for it to work. 

[triple back ticks]
stuff you want to say
[triple back ticks]

```
$ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```